src/OpenSpaceToolkit/Astrodynamics/Dynamics/AtmosphericDrag.cpp

```diff
@@ -29,9 +29,6 @@ using ostk::astrodynamics::trajectory::state::CoordinateSubset;
 using ostk::astrodynamics::trajectory::state::coordinatesubset::CartesianPosition;
 using ostk::astrodynamics::trajectory::state::coordinatesubset::CartesianVelocity;
 
-static const Derived::Unit GravitationalParameterSIUnit =
```

Comment:
This doesn't change anything, some of these were just defined in files they're not actually used in, so I went ahead and removed other examples like this.